esi_bot/commands.py

```diff
@@ -69,9 +69,9 @@ def hello(msg):
 
     if "whatup" in msg.args:
         return "not much. whatup{}".format(_fmt_speaker(msg))
-    elif msg.command in ("o7", "o/"):
+    if msg.command in ("o7", "o/"):
```

Comment:
these should be fine as `elif`s. if you're using bleeding edge pylint it recently experienced a regression for this. i've been pinning to `1.9.2` until the `2.x` line of pylint gets a little more stable